pages/docs/user-docs/build-busybox.md

```diff
@@ -0,0 +1,27 @@
+---
+title: busybox bootstrap module
+sidebar: user_docs
+permalink: build-busybox
+folder: docs
+toc: false
```

Comment:
these little pages probably don't need the TOC!

Reply:
It annoys me from a style perspective that some of our pages have text on top of toc, some don't, and some don't have toc.  I think it looks messy.  I am going for the same look and feel on every page:
1. few lines of text to intro the page
2. toc
3. main body of page
I think if we did this on every page it would make the site look much more unified and less piecemeal.

Reply:
If the TOC is one link it's not so useful. I'm ok if you want to change the style, but please try to keep text / objects off the page that aren't useful or necessary.